.gitignore

```diff
@@ -4,6 +4,7 @@
 /node_modules
 /.pnp
 .pnp.js
+pnpm-lock.yaml
```

Comment:
Eef... not sure if Google Cloud supports pnpm. It might be best to stick with npm for now.